frontend/src/app/footer/footer.component.html

```diff
@@ -1,7 +1,18 @@
-<div  *ngIf="status" class="footer">
-  <p>© 2019 BY THE RECTOR AND VISITORS OF THE UNIVERSITY OF VIRGINIA</p>
+<footer *ngIf="status" class="global-footer">
+
+  <a href="https://curry.virginia.edu/faculty-research/centers-labs-projects/supporting-transformative-autism-research-star" target="_blank">
+    <img src="/assets/logo/UVA_STAR-logo.svg" class="footer-logo">
+  </a>
+
+  <div class="footer-contact">
+    <address class="address">417 Emmet Street, Charlottesville, VA 22904</address>
+    <a href="tel:434-924-0742" class="tel">434-924-0742</a>
+    <a href="mailto:curry-admissions@virginia.edu" class="email">uvastar@virginia.edu</a>
```

Comment:
Is there a reason that these are different email addresses? The last feedback I see from the group says: The email address changed to: uvastar@virginia.org
The phone number changed to: 434)243-8747
(slide 28 in the feedback doc)